### PR TITLE
Fix linter

### PR DESCRIPTION
**Pull request by ericgaspar**

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/remove [hunk 1]
```diff
@@ -74,7 +74,7 @@ ynh_script_progression --message="Removing Tvheadend main directory and config f
 
 # Remove the app directory securely (/home/hts folder)
 ## Use rm -r because ynh_secure_remove says that /home/hts is not an acceptable path to delete
-rm -r $final_path
+ynh_secure_remove --file="$final_path"
 ynh_secure_remove --file="/etc/default/tvheadend"
```
Comment (sy6sy2) [inline scripts/remove]: Hi Eric, thank for the PR.
But as you can see on my comment here, `ynh_secure_remove` does not work in this case 😕

Comment (ericgaspar) [reply scripts/remove]: ok thanks, I missed your comment.
Indeed, `ynh_secure_remove` helper prevents from removing `/home/*` path...
[ynh_secure_remove](https://github.com/YunoHost/yunohost/blob/71a08c09ea5099d7951d8d7badf0ed3bc0a02cf2/data/helpers.d/utils#L715)

Comment (sy6sy2) [reply scripts/remove]: Maybe @alexAubin has an idea of a cleaner way to fix this linter warning?

Comment (sy6sy2) [reply scripts/remove]: ping @alexAubin

**Comment by yalh76** [comment]
!testme

**Pull request merged**